core/primitives/block/block.hpp

```diff
@@ -0,0 +1,59 @@
+#include <boost/optional.hpp>
+
+#include "primitives/address/address.hpp"
+#include "primitives/address/address_codec.hpp"
+#include "primitives/big_int.hpp"
+#include "primitives/cid/cid.hpp"
+#include "primitives/ticket/epost_ticket.hpp"
+#include "primitives/ticket/epost_ticket_codec.hpp"
+#include "primitives/ticket/ticket.hpp"
+#include "primitives/ticket/ticket_codec.hpp"
+
+namespace fc::primitives::block {
+  using primitives::BigInt;
+  using primitives::address::Address;
+  using primitives::ticket::EPostProof;
+  using primitives::ticket::Ticket;
+
+  // TODO(turuslan): FIL-72 signature
+  using Signature = std::vector<uint8_t>;
```

Comment:
is it a universal signature type? not pls?

Reply:
Lotus sources suggest so